python/pyspark/sql/connect/column.py

```diff
@@ -307,7 +305,21 @@ def __str__(self) -> str:
         return str(self.ref) + " ASC" if self.ascending else " DESC"
 
     def to_plan(self, session: "SparkConnectClient") -> proto.Expression:
-        return self.ref.to_plan(session)
+        # TODO: move SortField from relations.proto to expression.proto
```

Comment:
Nit: better to have a JIRA link here. I usually do TODO(SPARK-xxxx)

Reply:
nice, will update